SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -74,13 +75,19 @@ bool FStartDeployment::Update()
 				return;
 			}
 
-			FSpatialLaunchConfigDescription LaunchConfigDescription(AutomationWorkerType);
-			LaunchConfigDescription.SetLevelEditorPlaySettingsWorkerTypes();
+			FSpatialLaunchConfigDescription LaunchConfigDescription;
 
-			if (!GenerateDefaultLaunchConfig(LaunchConfig, &LaunchConfigDescription))
+			TMap<FName, FWorkerLaunchConfiguration> WorkerConfigMap;
+
+			FWorkerLaunchConfiguration Conf;
+			Conf.WorkerLoadBalancing = USpatialOSSingleWorkerStrategy::StaticClass()->GetDefaultObject<USpatialOSSingleWorkerStrategy>();
```

Comment:
are we starting local deployments by default with the single worker strategy? 
what happens if I change the config in the settings and then start the game? will the new strategy be applied or do I need to restart the deployment?

Reply:
This particular file is for the Tests.
Otherwise, yes, when load balancing is disabled, it's the SingleWorkerStrategy (times the offloaded workers).
If you change settings or the load balancing, you have to restart manually (I believe it's already the case, but I could try to improve that).

Reply:
https://improbableio.atlassian.net/browse/UNR-3332